new/reclassify_amazon_linux_platform_family.md

```diff
@@ -0,0 +1,30 @@
+---
+RFC: unassigned
+Author: Sean OMeara <sean@chef.io>
+Status: Draft
+Type: Standards
+---
+
+# Title
+
+Reclassify Amazon Linux platform_family
+
+## Motivation
+
+   As time passes, Amazon Linux looks less and less like a RHEL clone.
+   Release version numbers do not match EL5, EL6, EL7.
+   Amazon is now a rolling release
+   Different package sets are available
+   Package names and contents are different
+   Service providers are different
+
+## Specification
+
+Change Amazon Linux platform_family from 'rhel' to 'amazon'
```

Comment:
I want to get this right the first time... 

I guess it really comes down to the intent of the Amazon Linux maintainers on how far they plan on deviating from Fedora.

Reaching out...